AcadeMeData/tests/test_messages.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from AcadeMeData.models import Messages, User
+
+
+@pytest.mark.django_db
+class TestMessagesModel:
+    @pytest.fixture
+    def user_example(self):
+        user_data = {'username': "username2212", 'password': "password", 'email': "user@example.com", 'type': "S",
+                     'university': "RU",
+                     'degree': "CS"}
+        user = User.create_user(*user_data)
+        return user
+
+    def generate_message(self, msgID=1, userID=None, text='This is a test message yo'):
+        message = Messages(msgID=msgID, userID=userID, text=text)
```

Comment:
MHO, This should be a fixture also
The creation of user, messages should be in the fixture, you could call 2 fixtures if you like,

Reply:
Ok, will do.
I think you're right, generate_message should be a fixture.